src/main.cpp

```diff
@@ -26,47 +26,122 @@ using namespace boost;
 //
 
 CCriticalSection cs_setpwalletRegistered;
-set<CWallet*> setpwalletRegistered;
+set<CWallet *> setpwalletRegistered;
 
 CCriticalSection cs_main;
 
 CTxMemPool mempool;
 
-map<uint256, CBlockIndex*> mapBlockIndex;
-set<pair<COutPoint, unsigned int> > setStakeSeen;
+map<uint256, CBlockIndex *> mapBlockIndex;
+set<pair<COutPoint, unsigned int>> setStakeSeen;
 
 CBigNum bnProofOfStakeLimit(~uint256(0) >> 20);
 CBigNum bnProofOfStakeLimitV2(~uint256(0) >> 20);
 
-unsigned int nStakeMinAge = 8 * 60 * 60; // 8 hours
+unsigned int nStakeMinAge()
```

Comment:
@konez2k so here the int is just unsigned and defined as a variable... but is it okay that because I am returning an int at the end?

Reply:
nStakeMinAge is a variable, if you want to use a function you should declare it as 
unsigned int GetStakeMinAge();
and return the value of nStakeMinAge